### PR TITLE
Add change orientation callback

### PR DESCRIPTION
**Pull request by mustardandrew**
Added callback when changing the orientation of the screen.

### CODE REVIEW

**Comment by qWici** [comment]
This is an important addition to me. Please approve!

**Comment by georgebent** [comment]
Please approve this!
This is a very useful addition.

**Comment by matthewhudson** [comment]
@mustardandrew Sorry for the delay in reviewing, mind revisiting this? Happy to accept these changes given popular demand. 

Unfortunately, the `master` branch has drifted a bit since you originally opened this PR. I'm really sorry about that. On the bright side, the build process has been refined, I'm adding test coverage, and we now have ES5/ES6/UMD builds.

File: src/index.js [hunk 1]
```diff
@@ -266,12 +266,28 @@ handleOrientation = function () {
...
     addClass("landscape");
+    walkOnChangeOrientationList("landscape");
   } else {
     removeClass("landscape");
     addClass("portrait");
+    walkOnChangeOrientationList("portrait");
   }
 };
 
+changeOrientationList = [];
```
Comment (matthewhudson) [inline src/index.js]: Mind declaring this as a var at the top of the file?

Comment (mustardandrew) [reply src/index.js]: Declare changeOrientationList as a var at the top

**Pull request merged**